src/components/profile/ProfileService.js

```diff
@@ -253,8 +288,9 @@ goog.require('ga_urlutils_service');
           group.append('text')
               .attr('class', 'ga-profile-label ga-profile-label-x')
               .attr('x', width / 2)
-              .attr('y', height + options.margin.bottom - 2)
+              .attr('y', height - 4 + 3 * options.margin.bottom / 4)
```

Comment:
I did this so there is no overlapping between x-label and elevation labels when resizing the window.
![pic1](https://cloud.githubusercontent.com/assets/14076451/10507748/08f36760-7323-11e5-87ce-dae74b3cee7e.PNG)